aten/src/ATen/native/native_functions.yaml

```diff
@@ -6932,6 +6928,7 @@
   dispatch:
     ZeroTensor, CPU, CUDA, Meta, QuantizedCPU, QuantizedCUDA, MPS: view
     MkldnnCPU: mkldnn_view
+    NestedTensorCPU, NestedTensorCUDA: view_nested
```

Comment:
Doesn't need to this PR, but we should consider renaming the existing `DispatchKey::NestedTensor` to `DispatchKey::NestedTensorFunctionality`, and then making `DispatchKey::NestedTensor` a new alias key for "all nested tensor backends". That way you don't have special-case for cpu/cuda and in the other places.